.github/workflows/ci.yaml

```diff
@@ -51,7 +50,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-22.04]
-        ghc-version: ['9.8']
+        ghc-version: ["9.8"]
```

Comment:
Our [CONTRIBUTING.md](https://github.com/input-output-hk/ouroboros-leios/blob/6e7dcaf37c6de16365e0e83deb71ad26e0fce781/CONTRIBUTING.md?plain=1#L71) is out of sync: it recommends GHC 9.6.3. I think both versions probably work, but we might want to align the two files.